gradle/libs.versions.toml

```diff
@@ -1,10 +1,10 @@
 [versions]
-kotlin = "2.0.21"
+kotlin = "2.1.21"
 dokka = "2.0.0"
 nexusPublish = "2.0.0"
 npmPublish = "3.5.2"
 apiCompatibility ="0.17.0"
-kotlinxDatetime = "0.6.1"
+kotlinxDatetime = "0.7.1"
 kotlinJsWrappers = "1.0.0-pre.830"
 slf4jApi = "1.7.36"
 expressionParser = "1.2.0"
```

Comment:
Could you try updating to expression-parser `1.3.0-SNAPSHOT` and upgrading to kotlin `2.2.21`?